### ISSUE TITLE
Dropbear args not applied at boot

### ISSUE THREAD

**Issue by marcosjt7** [procount/pinn#702]
[README_PINN.md](https://github.com/procount/pinn/blob/master/README_PINN.md#securing-ssh) says to save the following in /settings/dropbear/dropbear to disable password authentication for SSH:
```
DROPBEAR_ARGS="-s -g"
```
However, even after rebooting, **password authentication is still possible**.
running `ps` shows
```
PID   USER     COMMAND
...
300 root     /usr/sbin/dropbear -R
```
Restarting dropbear manually successfully adds `DROPBEAR_ARGS` and disables password authentication:
```
root@recovery:~# /etc/init.d/S50dropbear restart
root@recovery:~# ps
PID   USER     COMMAND
...
 310 root     /usr/sbin/dropbear -s -g -R
```
But the problem resurfaces the next time the system reboots.

This suggests that `/settings/dropbear/dropbear` is not being linked to `/etc/default/dropbear` until _after_ `/etc/S50dropbear` is run by `/init`. That creates a security issue: the user may think they've locked down their Pi, but one can still SSH with the well-known default password.

**Comment by procount** [comment]
Thanks for identifying this issue. It does look like a sequence problem. I shall look into it

**Comment by procount** [comment]
This should now be fixed in v3.8.4
